userguide/recommender/coreml-deployment.md

```diff
@@ -18,15 +18,10 @@ my_recommender.export_coreml("MyRecommender.mlmodel")
 After you drag and drop the exported Core ML model in your iOS app, it will look something like this:
 ![Core ML Model Screenshot in Xcode](xcode-shot.png)
 
-Before you can go ahead and use the recommender model in your app, you must 
-create the `targets/libTuriCreate.dylib` file from the root directory of the 
-Turi Create repository by running the following command from your terminal:
-
-```bash
-./build_capi.sh -i -j 8 -r
-```
-
-After the C API build completes, you can drag and drop the `libTuriCreate.dylib` file from the `targets` directory of the root Turi Create repository into your Xcode project. 
+Before you can go ahead and use the recommender model in your app, you must
+drag and drop the `libRecommender-arm64.dylib` available on the 
```

Comment:
Maybe clarify which library is for which type of app? For a mac app, or for iOS Simulator, the x86_64 one would be used. For an iOS app, the arm64 one would be used.